ci-operator/templates/master-sidecar-3.yaml

```diff
@@ -104,7 +104,7 @@ objects:
           echo "Gathering artifacts ..."
         }
         trap 'teardown' EXIT
-        trap 'jobs -p | xargs -r kill; exit 0' TERM
+        trap 'jobs -p | xargs -r kill || true; exit 0' TERM
```

Comment:
This looks correct, can you change all the others as well (i.e. every usage of this in this repo)

Reply:
@smarterclayton updated, there were two kinds of changes:

```diff
-jobs -p | xargs -r kill
+jobs -p | xargs -r kill || true
```

```diff
-jobs -p | xargs kill || true
+jobs -p | xargs -r kill || true
```

Reply:
Hrm, I thought I got all the former.  Must have forked after my change.